tests/testthat/test-order-radix.R

```diff
@@ -56,11 +56,11 @@ test_that("all combinations of `direction` and `na_value` work", {
   )
   expect_identical(
     x[vec_order_radix(x, na_value = "largest", direction = "desc")],
-    x[order(x, na.last = TRUE, decreasing = TRUE)]
+    x[order(x, na.last = FALSE, decreasing = TRUE)]
```

Comment:
These tests confused me for a moment. They are a bit misleading because they imply a one-to-one correspondence between the two ordering behaviours. Should we test against `vec_order()` instead?

Also we should document in `?vec_order::na_value`  that it behaves differently from `order::na.last` (`na.last` doesn't interact with `decreasing` whereas `na_value` interacts with `direction`).